releases.md

```diff
@@ -0,0 +1,38 @@
+# Proposed Revised Repo Model and Release Plan
+
+The Node.js TSC has recently been discussing and hammering out the details
+for a revised general release plan. The fundamental details of the plan are:
+
+<img src="https://camo.githubusercontent.com/675f052419c56a583c19644d4da30cc0ff4e02bb/68747470733a2f2f636c6475702e636f6d2f6c4d48746245334e72782d3330303078333030302e706e67"/>
+
+The `master` branch remains as the target of primary development. The majority of commits generally land there. The overall goal is to keep master as stable as possible, with periodic, numbered `*-next.x` releases being cut at fairly regular intervals.
+
+Approximately every six months, a new *stable branch* is cut off master with an immediate `vX.0.0` release cut off that stable. Immediately after, the *semver-major* is bumped in `master`. In other words, if `master` is currently on `v3.0.0-next.x`, a new `v3.0.0` stable branch is cut off `v3.0.0-next.x` and `master` is bumped to `v4.0.0-next.x`. Six months later, a new `v4.0.0` stable branch is cut off `master` then `master` is bumped to `v5.0.0-next.x`.
+
+As mentioned, there will be fairly regular `next.x` releases off `master`. Developers can use these releases but ought to expect things to break once in a while. Breaking changes *will* land in `master` so it *is* possible (and *likely*) that breaking changes will occur from one `next.x` release to another. In particular, tools such as `NAN` may not yet have been updated to support the changes within any particular `next.x` release, and it's *likely* that native modules will not work.
```

Comment:
I suppose this is fine. Whatever we can do to message that people shouldn't go complaining about native modules not working with these versions.